products/jbrowse-react-linear-genome-view/src/corePlugins.ts

```diff
@@ -10,14 +10,14 @@ import Variants from '@jbrowse/plugin-variants'
 import Wiggle from '@jbrowse/plugin-wiggle'
 
 export default [
+  SVG,
   Alignments,
   BED,
   Config,
   DataManagement,
   GFF3,
   LinearGenomeView,
   Sequence,
-  SVG,
```

Comment:
motivation behind this re-arrangement?

Reply:
It matches what was done in `products/jbrowse-web/src/corePlugins.ts`, basically displays that don't have a specific renderer will default to the first registered one. This makes the default be the SVGFeatureRenderer.

Reply:
ah gotcha. i remember this being done in the jbrowse-web too. kind of weird to rely on this behavior in general but it does match jbrowse-web